README.md

```diff
@@ -19,17 +19,21 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Infrastructure and processes we build and/or support
 
+* [Ansible Community Forum](https://forum.ansible.com/): Official Ansible Discource forum.
```

Comment:
typo, DiscourSe

Reply:
yeah, i know though sometimes strange things happen to my not native brain:)

Reply:
fixed, thanks, merging